kythe/go/serving/pipeline/beamio/leveldb_test.go

```diff
@@ -106,3 +106,23 @@ func TestLevelDBSink(t *testing.T) {
 }
 
 func extendToKey(v beam.T) (beam.T, beam.T) { return v, v }
+
+func TestSchemaPreservingPathJoin(t *testing.T) {
+	exp := "gs://foo/bar/baz"
+	res := schemePreservingPathJoin("gs://foo", "bar/baz")
+	if exp != res {
+		t.Fatal("Expected [%s], got [%s]", exp, res)
```

Comment:
Please change `t.Fatal` to `t.Fatalf` throughout.

Reply:
Done